src/main/java/edu/harvard/iq/dataverse/GlobalId.java

```diff
@@ -226,6 +229,11 @@ private boolean checkDOIAuthority(String doiAuthority){
         
         return true;
     }
-    
-    
+
+    public static boolean hasDisallowedImportCharacters(String pidParam) {
+        Pattern p = Pattern.compile(DISALLOWED_CHARACTERS);
```

Comment:
This brings up the "enumerating badness" vs "enumerating goodness" debate.

Would it be a big complexity increase to flip the regex?

Reply:
I think I agree that we may be better served with a whitelist instead of a blacklist, at least to start. But I can see potential pitfalls, especially with supporting other languages.

Specifically, I wonder if there should be some restriction on the number of "/"s in the identifier